bin/deploy-vhost

```diff
@@ -825,9 +825,8 @@ sub update_config_and_schemas {
 sub run_tasks_from_config {
     my (%exec_extras) = @_;
     foreach my $exec_extra (@{$exec_extras{'user'}}) {
-        chdir_verbose(__LINE__, "$vhost_dir/$vcspath_install");
-        shell("su", $conf->{'user'}, "-c $exec_extra");
-    }
+        shell("su", "-", $conf->{'user'}, "-c cd \"$vhost_dir/$vcspath_install\" && $exec_extra");
```

Comment:
I'd be tempted to use `-l` rather than `-` (it doesn't look like the "last option" of the manpage actually applies, but might as well be safe).